app/views/main/index.html.erb

```diff
@@ -2,7 +2,7 @@
   <nav>
     <div class="title">
       <h1>Civic Tech Case Finder</h1>
-      <p>Case studies on civic tech topics</p>
+      <p>Case studies and project on different sectors in the civic tech space</p>
```

Comment:
I like that this tagline is more descriptive. It looks like there's an "s" missing "projects" and I'm wondering if the word "field" would work better than "space." What do you think?